src/parser/jobs/war/modules/PrimalProcs.tsx

```diff
@@ -0,0 +1,70 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITY_MISSED_PROCS = {
+	1: SEVERITY.MAJOR,
+}
+
+export class PrimalProcs extends CoreProcs {
+
+	override trackedProcs = [
+		{
+			procStatus: this.data.statuses.PRIMAL_REND_READY,
+			consumeActions: [this.data.actions.PRIMAL_REND],
+		},
+		{
+			procStatus: this.data.statuses.PRIMAL_RUINATION_READY,
+			consumeActions: [this.data.actions.PRIMAL_RUINATION],
+		},
+		{
+			procStatus: this.data.statuses.WRATHFUL,
+			consumeActions: [this.data.actions.PRIMAL_WRATH],
+		},
+	]
+
+	protected override addJobSpecificSuggestions() {
+		const missedRend = this.getDropCountForStatus(this.data.statuses.PRIMAL_REND_READY.id)
+		const missedRuination = this.getDropCountForStatus(this.data.statuses.PRIMAL_RUINATION_READY.id)
+		const missedWrath = this.getDropCountForStatus(this.data.statuses.WRATHFUL.id)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PRIMAL_REND.icon,
+			content: <Trans id="war.primalrend.suggestions.dropped.content">
+				Using <DataLink action="INNER_RELEASE" /> grants <DataLink status="PRIMAL_REND_READY" />. Use it for a strong attack and to unlock the even stronger <DataLink action="PRIMAL_RUINATION" />.
+			</Trans>,
+			tiers: SEVERITY_MISSED_PROCS,
+			value: missedRend,
+			why: <Trans id="war.primalrend.suggestions.dropped.why">
+				You missed <Plural value={missedRend} one="# Primal Rend use" other="# Primal Rend uses" />.
```

Comment:
_Nit:_ Phrasing might be a bit cleaner as:
`You missed <Plural value={missedRend} one="# use" other="# uses" /> of Primal Rend.`